README.md

```diff
@@ -3,8 +3,8 @@
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team02-w22-5pm-2/branch/main/graph/badge.svg?token=SsfDeOYjVv)](https://codecov.io/gh/ucsb-cs156-w22/team02-w22-5pm-2)
 
 Storybook is here:
-* Production: <https://ucsb-cs156-w22.github.io/starter-team02-docs/>
-* QA:  <https://ucsb-cs156-w22.github.io/starter-team02-docs-qa/>
+* Production: <https://github.com/ucsb-cs156-w22/team02-w22-5pm-2-docs/>
+* QA: <https://github.com/ucsb-cs156-w22/team02-w22-5pm-2-docs-qa/>
```

Comment:
Actually I think these are the wrong links. These are links to the github repos and not the storybook websites.